tests/unit/core/test_views.py

```diff
@@ -164,3 +164,14 @@ def test_clear_elements_with_stable(self, mock_object):
         self.viewset.request = request
         self.viewset.clear_elements(request=request, pk=MagicMock())
         self.assertEqual(len(mock_object.return_value.config), 1)
+
+
+class WebclientJavaScriptView(TestCase):
+    def setUp(self):
+        self.request = MagicMock()
+        self.view_instance = views.WebclientJavaScriptView()
+        self.view_instance.request = self.request
+
+    def test_app_configs(self):
+        response = self.view_instance.get()
```

Comment:
If I do this, django queries all permissions (e.g. see travis traceback): `for permission in Permission.objects.all()`.
This causes a `settings.DATABASES is improperly configured`-error. To execute the view, I have to mock everything where a DB query could occur, but this view is highly dynamic (finding plugins and special method in the app configs), that mocking all this is not good..